### PR TITLE
Enforces consistency with FontAwesome icons.

### PR DESCRIPTION
**Pull request by AnthonyBobsin**
Replaces all Bootstrap Glyphicon icons with the identical FontAwesome one. Directly related to fixing issue #1033.

### CODE REVIEW

**Comment by mmmavis** [comment]
hey @AnthonyBobsin , thanks for contributing and submitting this PR :+1: :smile_cat: Sorry for getting to this late. [The original ticket](https://github.com/mozilla/teach.webmaker.org/issues/1033) is kinda stale so I'll have to check with our designer and see if it's still valid.  Will get back to you!

**Comment by AnthonyBobsin** [comment]
hi @mmmavis, anytime, love the project! Sounds good :+1:

**Comment by Pomax** [comment]
If it's not too much work, could we turn `<i>` back into `<span>`, with an eye on accessibility? (these icons aren't italicized text =)

File: pages/clubs.jsx [hunk 1]
```diff
@@ -39,11 +39,11 @@ var ClubListItem = React.createClass({
       ownerControls = (
         <p>
           <button className="btn btn-default btn-xs" onClick={this.props.onEdit.bind(null, club.url)}>
-            <span className="glyphicon glyphicon-pencil"></span> Edit
+            <i className="club-button fa fa-pencil"></i> Edit
```
Comment (Pomax) [inline pages/clubs.jsx]: since we're in JSX, we can use the selfclosing syntax here, and in L46 too

File: components/map.jsx [hunk 2]
```diff
@@ -131,13 +131,13 @@ var MarkerPopupClub = React.createClass({
         <div className="action-panel">
           <button className="btn btn-default btn-xs"
            data-club-action="edit" data-club-url={this.props.url}>
-            <span className="glyphicon glyphicon-pencil"></span> Edit
+            <i className="club-button fa fa-pencil"></i> Edit
```
Comment (Pomax) [inline components/map.jsx]: since we're in JSX, we can use the selfclosing syntax here, and in L140 too

File: components/login.jsx [hunk 3]
```diff
@@ -131,7 +131,7 @@ var Login = React.createClass({
     return (
       <div>
         <br/>
-        <span className="glyphicon glyphicon-wrench"></span>
+        <i className="fa fa-wrench"></i>
```
Comment (Pomax) [inline components/login.jsx]: since we're in JSX, we can use the selfclosing syntax here

**Comment by Pomax** [comment]
@AnthonyBobsin awesome! Thanks for all the help!

**Comment by AnthonyBobsin** [comment]
I've made all the changes that you told me. Please let me know if there are still any issues

**Comment by Pomax** [comment]
closing because the codebase kind of passed this PR by, and there's a design audit in process cross-moz-foundation which includes making a decision on icons.

**Pull request closed without merging**